Description: do not link against libm unnecessarily
 (Bare.so uses none of the libraries symbols)
Author: Florian Schlichting <fsfs@debian.org>

--- a/Makefile.PL
+++ b/Makefile.PL
@@ -66,7 +66,6 @@
 }
 sub gen_cc {
   WriteMakefile( @basics,
-    LIBS      => ['-lm'],
     OBJECT    => 'Bare.o parser.o',
     LDDLFLAGS => '-shared -L/usr/local/lib',
   );
